src/components/document/document.tsx

```diff
@@ -55,17 +55,14 @@ const DownloadButton = ({ onClick }: { onClick: SVGClickHandler }) => {
 const PublishButton = ({ onClick, dataTestName }: { onClick: () => void, dataTestName?: string }) => {
   return (
     <IconButton icon="publish" key="publish" className="action icon-publish" dataTestName={dataTestName}
-                onClickButton={onClick} title="Publish Workspace" />
+                onClickButton={onClick} title="Publish To Workspace" />
```

Comment:
Actually, this seems wrong to me. The first is "Publish Workspace" as in, "Publish This Workspace" or "This Document". That's what the button does. It performs the action "Publish to Class Work" if we want to label the target. That's a little dangerous in case we change the name Class Work to Class Activities or something. Am I not understanding where these labels go?

Reply:
The `title` refers to the tooltip that is displayed when hovering over the button. I'm fine with leaving it as `Publish Workspace` as well. My only point is that `Publish To Workspace` doesn't make sense.

Reply:
Right, so this should go back to its original language. The Tooltip should read "Publish Workspace".